Lib/fontTools/t1Lib/__init__.py

```diff
@@ -367,3 +513,72 @@ def stringToLong(s):
 	for i in range(4):
 		l += byteord(s[i]) << (i * 8)
 	return l
+
+
+# PS stream helpers
+
+font_dictionary_keys = copy.copy(_type1_pre_eexec_order)
+# t1write.c:writeRegNameKeyedFont
+# always counts following keys
+font_dictionary_keys.remove("FontMatrix")
+
+FontInfo_dictionary_keys = copy.copy(_type1_fontinfo_order)
+# extend because AFDKO tx may use following keys
+FontInfo_dictionary_keys.extend([
+	"FSType",
+	"Copyright",
+])
+
+Praivate_dictionary_keys = [
+	# We don't know what names will be actually used.
+	# "RD",
+	# "ND",
+	# "NP",
+	"Subrs",
+	"OtherSubrs",
+	"UniqueID",
+	"BlueValues",
+	"OtherBlues",
+	"FamilyBlues",
+	"FamilyOtherBlues",
+	"BlueScale",
+	"BlueShift",
+	"BlueFuzz",
+	"StdHW",
+	"StdVW",
+	"StemSnapH",
+	"StemSnapV",
+	"ForceBold",
+	"LanguageGroup",
+	"password",
+	"lenIV",
+	"MinFeature",
+	"RndStemUp",
+]
+
+# t1write_hintothers.h
+hintothers = \
+	"/OtherSubrs[{}{}{}{systemdict/internaldict known not{pop 3}{1183615869\n" + \
+	"systemdict/internaldict get exec dup/startlock known{/startlock get exec}{dup\n" + \
+	"/strtlck known{/strtlck get exec}{pop 3}ifelse}ifelse}ifelse}executeonly]def"
+# t1write.c:saveStdSubrs
+def bytechrjoin(iterable, joiner=b""):
+	return tobytes(joiner).join(bytechr(item) for item in iterable)
+std_subrs = [
+	# 3 0 callother pop pop setcurrentpoint return
+	bytechrjoin([0x8e, 0x8b, 0xc, 0x10, 0xc, 0x11, 0xc, 0x11, 0xc, 0x21, 0xb]),
```

Comment:
Why not use byte literals, like you do below?

Reply:
OK. I copied them from [t1write.c](https://github.com/adobe-type-tools/afdko/blob/develop/c/shared/source/t1write/t1write.c#L590-L605) and used them, which is why they look like this.